### PR TITLE
Fix gas prices RPC

### PR DESCRIPTION
**Pull request by Valentine1898**

### CODE REVIEW

File: packages/services/src/view-service/gas-prices.ts [hunk 1]
```diff
@@ -24,13 +24,13 @@ export const gasPrices: Impl['gasPrices'] = async (_, ctx) => {
...
-  const altGasPRices = await indexedDb.getAltGasPrices();
+  const altGasPrices = await indexedDb.getAltGasPrices();
   if (!gasPrices) {
     throw new ConnectError('Gas prices is not available', Code.NotFound);
   }
 
   return {
     gasPrices,
-    altGasPRices,
+    altGasPrices,
```
Comment (Valentine1898) [inline packages/services/src/view-service/gas-prices.ts:34]: Shouldn't that have resulted in a type error?

**Comment by VanishMax** [comment]
Whoa, I was just trying to understand why it is always empty. Checking if that helps

**Comment by Valentine1898** [comment]
> No type error since we don't type the returns from the services and then use the types from ViewService protobuf – a mistake we have to fix at some point

That's terrible, we definitely need to take care of that

**Comment by Valentine1898** [comment]
I'll hold off on this PR since @TalDerei  is going to provide more clarification

**Review by TalDerei — APPROVED** [review]

**Comment by Valentine1898** [comment]
> @Valentine1898 The fixedAltGasPrices struct in the governance proposal was crafted incorrectly with a bracket misalignment, so the prices were interpreted as unknown fields. A new proposal should be submitted with the parameters properly aligned and set correctly.

Ohh 🤯
But it doesn't actually block Max's PR https://github.com/penumbra-zone/web/pull/1614

**Comment by grod220** [comment]
We should return something like:

```ts
return new GasPricesResponse({
    gasPrices,
    altGasPrices,
  });
```

for all services to get typesafety. Will open another issue for this.

**Pull request merged**